arlebnispfade/src/components/HeaderComp.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+    <div class="header" id="header">
+      <div class="header-content">
+        <div v-if="BackButtonLink == null && BackButtonMethod == null" style="width: 40px;"></div>
+        <router-link class="back-button" v-if="BackButtonLink != null" :to="BackButtonLink"><img src="@/assets/icons/arrow_back.svg"></router-link>
+        <div class="back-button" v-if="BackButtonMethod != null" @click="$emit(BackButtonMethod)"><img src="@/assets/icons/arrow_back.svg"></div>
+        <h1>{{title}}</h1>
+        <div style="width: 40px;"></div>
+      </div>
+    </div>
+</template>
+
+<script>
+export default {
+  mounted() {
+  },
+  props: {
+    title: String,
+    BackButtonLink: String,
+    BackButtonMethod: String
+  },
+  methods: {
+  }
+};
+</script>
+
+<style scoped>
+.header {
+  background: var(--primary-color);
+  height: 7vh;
+  width: 100vw;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  position: sticky;
+  top: 0;
+  z-index: 100;
+}
+
+.header-content {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-content: center;
+  flex-wrap: wrap;
+  height: 100%;
+  width: 98%;
+}
+
+.button {
+  align-items: center;
+}
+
+h1 {
+  font-size: 1.2rem;
+  color: #ffffff;
+  margin: 0;
+  align-self: center;
+  font-family: "Open Sans", Verdana, Geneva, sans-serif;
```

Comment:
ich weiß nicht ob du solche werte auch als Variablen anlegen willst. Ist denke ich mal eine Abwägung in der Projekt-Lebensdauer..

Reply:
Guter Punkt, ich habe Eigenschaften, welche nicht Header-spezifisch waren und auch an anderen Stellen vorkommen, via CSS-Variablen implementiert.